tools/topology/sof/pipe-dai-capture.m4

```diff
@@ -20,3 +20,5 @@ W_PIPELINE(N_DAI_IN, SCHEDULE_DEADLINE, SCHEDULE_PRIORITY, SCHEDULE_FRAMES, SCHE
 
 P_GRAPH(DAI_NAME, PIPELINE_ID,
 	LIST(`		', `dapm(DAI_BUF, N_DAI_IN)'))
+
+indir(`define', concat(`PIPELINE_SCHED_COMP_', PIPELINE_ID), N_DAI_IN)
```

Comment:
@keyonjie this should be a separate commit

Reply:
OK, let me add a commit with it as the first one.